src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2.py

```diff
@@ -243,8 +243,8 @@ def arbeitsl_geld_2_regelsatz_m_bg_bis_2010(
 
     """
     # Note that we, currently, do not support households with more than 2 adults.
-    weitere_erwachsene = max(anz_erwachsene_hh - 2, 0)
```

Comment:
Is the comment above still relevant now that we have Bedarfsgemeinschaften? And how could we have additional grown-ups in a Bedarfsgemeinschaft?

Reply:
So the law for this function says the following:

> Haben zwei Angehörige der Bedarfsgemeinschaft
> das 18. Lebensjahr vollendet, beträgt die Regelleistung
> jeweils 90 vom Hundert der Regelleistung nach Absatz 2.
> Die Regelleistung für sonstige erwerbsfähige Angehörige
> der Bedarfsgemeinschaft beträgt 80 vom Hundert der
> Regelleistung nach Absatz 2.

The additional grown-ups could be children of the parents. I don't see why the limitation to two adults in the Bedarfsgemeinschaft should be relevant anymore.

Reply:
Cool, thanks! I was just confused then.